### PR TITLE
Update to 1.19

### PR DESCRIPTION
**Pull request by BlueGamingCat1**
Updated to 1.19!

### CODE REVIEW

File: src/main/java/pro/mikey/autoclicker/AutoClicker.java [hunk 1]
```diff
@@ -216,10 +216,10 @@ private void keyInputEvent(MinecraftClient mc) {
         while (toggleHolding.wasPressed()) {
             this.isActive = !this.isActive;
             mc.player.sendMessage(
-                    (this.isActive ? Language.MSG_HOLDING_KEYS : Language.MSG_RELEASED_KEYS)
-                            .getText()
-                            .formatted(this.isActive ? Formatting.GREEN : Formatting.RED),
-                    true);
+                    (this.isActive ? Language.MSG_HOLDING_KEYS : Language.MSG_RELEASED_KEYS).getText(), true
```
Comment (MichaelHillcox) [inline src/main/java/pro/mikey/autoclicker/AutoClicker.java:219]: Does this mean you're not using the colors anymore?

**Review by MichaelHillcox — CHANGES_REQUESTED** [review]
Otherwise looks good

File: src/main/java/pro/mikey/autoclicker/AutoClicker.java (hunk 1 above)
Comment (BlueGamingCat1) [reply src/main/java/pro/mikey/autoclicker/AutoClicker.java:219]: It does, I can go back and fix it but didn't see much of an issue with it being gone.

Comment (MichaelHillcox) [reply src/main/java/pro/mikey/autoclicker/AutoClicker.java:219]: Just user feedback, I'd prefer it is there, the MutableComponent (not sure what it is in yarn) should be pretty simple to add it back to.

Comment (BlueGamingCat1) [reply src/main/java/pro/mikey/autoclicker/AutoClicker.java]: Alright fixed it, was quite simple like you said!

**Review by MichaelHillcox — APPROVED** [review]

**Pull request merged**